gcloud/pubsub/message.py

```diff
@@ -15,6 +15,11 @@
 """Define API Topics."""
 
 import base64
+import datetime
+
+import pytz
+
+RFC3369 = '%Y-%m-%dT%H:%M:%S.%fZ'
```

Comment:
1. Yes, 3339 is correct.
2. I'm ambivalent about the public bit.
3. Sharing is tricky -- we don't want to introduce cycles.  It could maybe be in `gcloud._helpers`?